Cargo.lock

```diff
@@ -822,9 +822,9 @@ checksum = "0ab1bc2a289d34bd04a330323ac98a1b4bc82c9d9fcb1e66b63caa84da26b575"
 
 [[package]]
 name = "ordered-float"
-version = "2.10.0"
+version = "3.4.0"
```

Comment:
Of course the funny thing about this is https://github.com/10XGenomics/fastq_set/pull/143

Reply:
Ha, nice catch, thanks!